website/src/routes/covid.ts

```diff
@@ -30,7 +31,7 @@ class CovidConstants {
 
     public readonly organism = Organisms.covid as typeof Organisms.covid;
     public readonly locationFields = ['region', 'country', 'division'];
-    public readonly defaultDateRange: DateRange = 'last6Months';
+    public readonly defaultDateRange: DateRange = DateWindows.last6Months;
```

Comment:
Actually `DateRange` and `DateWindow` are two different types. They just happen to have overlapping values. I'm not sure whether it's a good idea to use this constant here.